pro/src/components/CollectiveOfferNavigation/__specs__/CollectiveOfferNavigation.spec.tsx

```diff
@@ -292,4 +294,118 @@ describe('CollectiveOfferNavigation', () => {
 
     expect(notifyError).toHaveBeenCalledWith('Impossible de dupliquer l’image')
   })
+
+  it('should archive an offer template', async () => {
+    renderCollectiveOfferNavigation({
+      ...props,
+      isTemplate: true,
+      isCreatingOffer: false,
+      isArchivable: true,
+    })
+
+    const archiveButton = screen.getByRole('button', {
+      name: 'Archiver',
+    })
+
+    await userEvent.click(archiveButton)
+    const confirmArchivingButton = screen.getByText('Archiver l’offre')
+    await userEvent.click(confirmArchivingButton)
+
+    expect(notifySuccess).toHaveBeenCalledWith(
+      'Une offre a bien été archivée',
+      {
+        duration: 8000,
+      }
+    )
+  })
+
+  it('should archive an offer bookable', async () => {
+    renderCollectiveOfferNavigation({
+      ...props,
+      isTemplate: false,
+      isCreatingOffer: false,
+      isArchivable: true,
+    })
+
+    const archiveButton = screen.getByRole('button', {
+      name: 'Archiver',
+    })
+
+    await userEvent.click(archiveButton)
+    const confirmArchivingButton = screen.getByText('Archiver l’offre')
+    await userEvent.click(confirmArchivingButton)
+
+    expect(notifySuccess).toHaveBeenCalledWith(
+      'Une offre a bien été archivée',
+      {
+        duration: 8000,
+      }
+    )
+  })
+
+  it('should not see archive button if offer already archived', () => {
```

Comment:
(suggestion) en soit là on ne sait pas que l'offre a déjà été archivée, moi je dirais de rester moins spécifique en mettant 'should not see archive button when offer is not archivable' surtout qu'à terme ce sera le back qui nous dira si oui ou non on peut archiver